install/install_windows.bat

```diff
@@ -1 +1,2 @@
-plink -ssh -v -pw raspberry pi@192.168.0.31 -m setup.sh
\ No newline at end of file
+plink -ssh -v -pw raspberry pi@192.168.0.31 -m setup.sh
+:: this file needs review as i'm sure that the hardcoded IP isnt going to work for everyone
```

Comment:
This wouldn't be the place to review and the instructions say to edit the file.

Reply:
very fair, just looked in the file to make sure everything made sense for windows. no reason to include this commit